lib/ontology_parsing_worker.rb

```diff
@@ -0,0 +1,14 @@
+class OntologyParsingWorker < Worker
+  sidekiq_options retry: 3, queue: 'hets'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.